eve/main.yml

```diff
@@ -55,3 +68,38 @@ stages:
         command: make lint
         flunkOnFailure: false
         haltOnFailure: false
+
+  single-node:
+    worker:
+      type: openstack
+      image: CentOS 7 (PVHVM)
+      flavor: io1-30
```

Comment:
`io1-30`? any particular reason why?

Reply:
Nope, copy-pasted from metalk8s/1.3. Any advice?

Reply:
no advice, but more like a question, how many ressources do you think you need?

This flavor gives you a VM with 30Gb of memory. If you think that this is what you need, go for it. Otherwise I suggest you select another flavor.

Reply:
platform9 flavors are supported here too no ?

Reply:
Chose `m1.medium` (9bc5a0c02c36f276a0c87d60afcd7a5bce352121)